### ISSUE TITLE
Chromebook emulation

### ISSUE THREAD

**Issue by Fgfan** [nbarkhina/N64Wasm#6]
When I tried this out on my laptop, it was pretty smooth on majoras's mask. But when i switched to chromebook, the results were less than stellar. It was constantly dropping at all times to 5-10 fps every time I move. Can you try to make it at least playable for the chromebook and other older devices?

**Comment by nbarkhina** [comment]
I don't have a chromebook to test on, but it's probably the hardware itself and not the chrome OS. Currently you do need a pretty beefy machine to run this emulator.

**Comment by slambo311** [comment]
Well, my chromebook can play that one Half-Life web version with max settings and still run at 60 fps, but it's still between 5-15 fps on mario 64.

**Comment by slambo311** [comment]
I highly doubt it's hardware. Chrome OS is pretty crappy, but my Chromebook's beefy enough to play Minecraft at 32 render distance, and lags like hell on here.

**Comment by TheGhostboi64** [comment]
> I highly doubt it's hardware. Chrome OS is pretty crappy, but my Chromebook's beefy enough to play Minecraft at 32 render distance, and lags like hell on here.

I can play sm64.z64 really fine on a chromebook.

**Comment by slambo311** [comment]
Yeah, well older versions exist. Mine doesn't even have a touchscreen.

**Comment by TheGhostboi64** [comment]
> Yeah, well older versions exist. Mine doesn't even have a touchscreen.

Mine doesn't either.

**Comment by slambo311** [comment]
Alright, touchscreens aren't exactly new anyway but still I have a pretty old Chromebook.
